tests/tests/regional_atmaq

```diff
@@ -100,6 +100,6 @@ export CCPP_SUITE=FV3_GFS_v15p2
 
 if [[ $MACHINE_ID = hera ]]; then
   TPN=30
-elif [[ $MACHINE_ID = cheyenne || $MACHINE_ID = gaea ]]; then
+elif [[ $MACHINE_ID = cheyenne || $MACHINE_ID = gaea* ]]; then
```

Comment:
Wanted to confirm that these TPN changes are still necessary on c5?

Reply:
they were still needed based on my testing (tests failed without them)

Reply:
TPN=128 should be set for Gaea C5. 
Settings for Gaea C5 could be done similar to those for wcoss2

Reply:
Gaea TPN=36 (Gaea C4)

Reply:
@natalie-perlin please see https://github.com/ulmononian/ufs-weather-model/blob/8567959bece0cdbbeb879c2308b581f7afd83624/tests/default_vars.sh#L216-L218 for the default TPN settings. for the specific regional atmaq tests, the TPN value needs changed.

Reply:
Oh, OK, thanks, @ulmononian !